scripts/linux-prereqs.sh

```diff
@@ -150,28 +150,23 @@ elif type apt-get > /dev/null 2>&1; then
 
     if [ $NETCOREDEPS -ne 0 ]; then
         checkNetCoreDeps aptSudoIf "install -yq libicu[0-9][0-9] libkrb5-3 zlib1g"
-        # Determine which version of libssl to install
+        
+        # Check for openssl
         # dpkg-query can return "1" in some distros if the package is not found. "2" is an unexpected error
-        LIBSSL=$(dpkg-query -f '${db:Status-Abbrev}\t${binary:Package}\n' -W 'libssl1\.0\.?' 2>&1)
+        LIBSSL=$(dpkg-query -f '${db:Status-Abbrev}\t${binary:Package}\n' -W 'openssl' 2>&1)
         if [ $? -eq 2 ]; then
            echo "(!) Failed see if libssl already installed!"
```

Comment:
`openssl`, not `libssl`

Reply:
The script is using `openssl` now. Is this a change request?

Reply:
On line 158, in this code `echo "(!) Failed see if libssl already installed!"` Shouldn't it be `openssl` in stead of `libssl`?

Reply:
Yeah, I made the change, sorry for the oversight. This thread can be resolved as it is outdated.